### PR TITLE
reefknot + wagmi update

### PR DESCRIPTION
**Pull request by kozlovzxc-at-lido**
### Description

Updated reefknot & wagmi to the latest version

Also fixes UI-770

### Demo

image

### Code review notes

- moved .env to .env.sample, so it never commited
- one eslint rule wasn't working with reef-knot, so. I disabled it for specific import

### Testing notes

Check if all operations with wallet are working

### Checklist:

- [x] Checked the changes locally.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ const tsRules = {
 
 // Import
 const importRules = {
-  'import/no-unresolved': ['error', { ignore: ['.svg'] }],
+  'import/no-unresolved': ['error', { ignore: ['.svg', 'reef-knot'] }],
```
Comment (DiRaiks) [inline .eslintrc.js:58]: Why is this needed?

**Comment by Jeday** [comment]
image
Error at build

**Review by alx-khramov — APPROVED** [review]

**Review by Jeday — APPROVED** [review]

**Review by manneredboor — APPROVED** [review]
